org.eclipse.winery.repository.rest/src/test/resources/entitytypes/artifacttypes/instance.xml

```diff
@@ -0,0 +1,14 @@
+<tosca:Definitions xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:tosca="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns1="http://www.eclipse.org/winery/model/selfservice" id="winery-defs-for_ns0-DressageEquipment_Pony" targetNamespace="http://winery.opentosca.org/test/ponyuniverse">
```

Comment:
The file name does not reflect its content. It is the "DressageEquipment_Pony" artifact type. Thus, it should be called `DressageEquipment_Pony.xml`.

Reply:
This comment applies TO ALL OTHER instance.xml files.